packages/react-components/babel-preset-storybook-full-source/package.json

```diff
@@ -26,13 +26,9 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
+    "@babel/core": "^7.10.4",
+    "prettier": "2.2.1",
+    "pkg-up": "^3.1.0",
     "tslib": "^2.1.0"
-  },
-  "exports": {
```

Comment:
let's keep to export (to be aligned with others) but lets use for `require` the same as main